cuesubmit/cuesubmit/Submission.py

```diff
@@ -72,8 +73,14 @@ def buildBlenderCmd(layerData):
         renderCommand += ' -o {}'.format(outputPath)
     if outputFormat:
         renderCommand += ' -F {}'.format(outputFormat)
-    # The render frame must come after the scene and output
-    renderCommand += ' -f {frameToken}'.format(frameToken=Constants.FRAME_TOKEN)
+    if frameRange:
+        # Renders animation within frame range
+        renderCommand += (' -s {startFrame} -e {endFrame} -a'
```

Comment:
Could you add a comment explaining what the `-a` flag is for?

Reply:
Resolved in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1337/commits/2d254c7ea900bc3babafb9871040b262ec6a7033